src/applications/mhv-secure-messaging/util/constants.js

```diff
@@ -410,7 +410,9 @@ export const BreadcrumbViews = {
   MOBILE_VIEW: 'mobile-view vads-u-margin-left--neg3 vads-u-padding-y--0p5',
 };
 
+// When RemoveLandingPage work is complete, replace the following constants with the NEW_ versions
```

Comment:
suggest including the flag name in comments here too. So that when we do a cleanup and search for flag name instances in code, we come across this part